### PR TITLE
Add instructions about using nlohmann/json with the conda package manager

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,6 +165,8 @@ If you are using [CocoaPods](https://cocoapods.org), you can use the library by
 
 If you are using [NuGet](https://www.nuget.org), you can use the package [nlohmann.json](https://www.nuget.org/packages/nlohmann.json/). Please check [this extensive description](https://github.com/nlohmann/json/issues/1132#issuecomment-452250255) on how to use the package. Please files issues [here](https://github.com/hnkb/nlohmann-json-nuget/issues).
 
+If you are using [conda](https://conda.io/), you can use the package [nlohmann_json](https://github.com/conda-forge/nlohmann_json-feedstock) from [conda-forge](https://conda-forge.org) executing `conda install -c conda-forge nlohmann_json`. Please files issues [here](https://github.com/conda-forge/nlohmann_json-feedstock/issues).
```
Comment (nlohmann) [inline README.md]: Typo: "Please files issues" -> "Please file issues"

**Comment by nlohmann** [comment]
Thanks for the PR. I have two questions:

- Can you please provide an example of the required steps to compile a simple hello-world program using the library and conda? I do not use package managers myself, so it's hard for me to understand how easy the integration is. This should not go into the README - an example for me is sufficient as I am overworking the website where I shall have more space to discuss the different package managers.
- Does the page automatically use the most recent version of the library or is a manual step required? I am asking because I need to know if I need to notify in case of a new release.

**Comment by tadeu** [comment]
Hi @nicoddemus and @nlohmann, I've created a short example of the required steps to compile a simple hello-world program using the library and conda, it's here: https://github.com/tadeu/hello_json/

**Comment by nicoddemus** [comment]
Cool, thanks @tadeu

File: README.md (hunk 1 above)
Comment (nicoddemus) [reply README.md]: Oops fixed.

**Comment by nicoddemus** [comment]
> Does the page automatically use the most recent version of the library or is a manual step required? I am asking because I need to know if I need to notify in case of a new release.

conda-forge has a bot which automatically updates the package within a few hours after a new release, so no notification from your part will be necessary. 👍

**Comment by nlohmann** [comment]
Thanks a lot!

**Comment by nicoddemus** [comment]
Awesome, thanks!

**Pull request merged**